CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-#  3.0.0-rc.5 (2018-11-19)
+# 3.0.0-rc.6 (2018-11-30)
+
+- Fixed modal focus to ignore visually hidden elements. [#12](https://github.com/blackbaud/skyux-modals/pull/12)
```

Comment:
Do we need "visually" here? It's not that important, but it seems kinda weird. I mean, are there other ways that elements would be hidden other than visually? If so, I guess we need the word here. But if not, it seems redundant.

Reply:
Yeah not really. I'll take it out